softwareComponents/gazebosim/common/pidController.hpp

```diff
@@ -32,8 +32,8 @@ class Buffer
 
         auto old = _buffer[ _pos ];
         _buffer[ _pos ] = value;
```

Comment:
Tady bude ještě warning...
Bych změnil ten _pos na size_t... já vím proč tam je int, ale asi jsem změnil názory..

Reply:
`int` je hlavně blbě. Chceme `ssize_t`, který tedy standard nedefinuje a místo toho používá `ptrdiff_t`. Jsem pro v nějaké další vlně vyrobit `using ssize_t = ptrdif_t` a používat to pro indexaci a velikosti.

Reply:
Kažpodádně `_buffer[ _pos ]` háže warning, nebo ne? Tak aspoň `to_unsigned` by to chtělo

Reply:
Neháže.